test/test_ops.py

```diff
@@ -229,27 +237,29 @@ def test_variant_consistency_eager(self, device, dtype, op):
                           (dtype.is_floating_point or op.supports_complex_autograd))
         samples = op.sample_inputs(device, dtype, requires_grad=_requires_grad)
         for sample in samples:
+            # Check grad only on the first input Tensor
+            tensor = sample.input if isinstance(sample.input, torch.Tensor) else sample.input[0]
```

Comment:
`# TODO`?

Reply:
This is already doing what the comment says, for a TensorList input it will only check the grad of the first Tensor in the list. Although now that I think I should make sure to pick the first Tensor in the list that requires grad.

Reply:
That seems like a refinement for the future.

The "TODO" might be to check the grad for each tensor in the list that requires grad.